decision_service/dsclientlib/ds_restapi_data_transport.cc

```diff
@@ -0,0 +1,10 @@
+#include "ds_restapi_data_transport.h"
+
+namespace decision_service { namespace model_mangement {
+
```

Comment:
Is this a stub to be filled in?

Reply:
As part of model management PR (not in scope for the logging PR)

---
In reply to: [187104321](https://github.com/JohnLangford/vowpal_wabbit/pull/1479#discussion_r187104321) [](ancestors = 187104321)